module-install-many.ks.in

```diff
@@ -6,18 +6,17 @@
 %ksappend common/common_no_payload.ks
 
 %packages
-@^Fedora Server Edition
 @nodejs:10/default
 @django:1.6/default
 # the postgress module has no default profile,
 # so we set some explicitly here
 @postgresql:9.6/server
 @postgresql:9.6/client
 # according to modulemd default "server" should be the default profile
-@mariadb:10.3/server
-@meson:latest/default
-@swig:3.0/default
-@gimp:2.10/default
+@mariadb:10.4/server
+@perl:5.32/minimal
+@swig:4.0/default
+@ruby:master/default
```

Comment:
Do we need a nightly test that installs so many modules of specific versions? I am afraid that it will start to fail in a few weeks again.

Reply:
Yeah, lets cut it down a bit.